client/src/modules/list/model/reducer.js

```diff
@@ -112,6 +112,15 @@ const lists = (state = {}, action) => {
         [action.payload.listId]: items(currentList, action)
       };
     }
+    case ListActionTypes.FAVOURITES_SUCCESS: {
+      return {
```

Comment:
```
const { payload: { isFavourite, listId } } = action; 
```
and then instead of:
```
isFavourite: action.payload.isFavourite
```
just use:
```
isFavourite
```